ArticlesApp/ArticlesApp/Managers/DataManager.swift

```diff
@@ -4,33 +4,50 @@ enum DataManagerError: Error {
     case Unknown
     case FailedRequest
     case InvalidResponse
+    case SerializationError
```

Comment:
Swift 4.0 says to use first letter lowercased as a standard now for enum values. So `case serializationError` instead. Same for the other values :)